wiki/Community/Profile_badge/en.md

```diff
@@ -0,0 +1,40 @@
+---
+tags:
+  - badges
+  - profile badges
+  - user badge
+  - user badges
+---
+
+# Profile badge
+
+*Not to be confused with [group](/wiki/Community/Group) badges, which also appear on user profiles.*\
+*For rules regarding usage as tournament prizing, see: [Official tournament support § Profile Badges](/wiki/Tournaments/Official_support#profile-badges)*
+
+![Screenshot of HappyStick's badges on the website](img/HappyStick.png "Profile badges display under the avatar and username on the website.")
+
+**Profile badges** (commonly referred to as just *badges*) are small graphics on user profile pages awarded for various achievements. They are most commonly given as [tournament](/wiki/Tournaments) and [contest](/wiki/Contests) prizes, but have other uses, such as rewards for [community contributors](/wiki/People/Community_Contributors), [Beatmap Spotlights](/wiki/Beatmap_Spotlights#rewards), and continued [group](/wiki/Community/Group) membership.
+
+When hovering over a badge, a tooltip is shown with more details about why the badge was awarded.
+
+## Tournament seeding
+
+*Main page: [Badge-weighted seeding](/wiki/Tournaments/Badge-weighted_seeding)*
+
+In the [game client](/wiki/Client) and on the website, badges are purely decorative. However, due to [tournament](/wiki/Tournaments) prize badges showcasing players' performance, some tournaments have adopted [seeding](https://en.wikipedia.org/wiki/Seed_(sports)) methods that factor in the amount of badges players have, commonly known as [badge-weighted seeding](/wiki/Tournaments/Badge-weighted_seeding) (BWS).
+
+## Old website
```

Comment:
```suggestion
## History
```

Reply:
i feel like this section title implies it would be more comprehensive, but it's actually just a quick explanation of the look on the new-old website and random userpage height fact

Reply:
I think I prefer history since it will likely be expanded in the future if/as more changes are made.